src/components/accessories/laboratory/LaboratoryDetails.tsx

```diff
@@ -0,0 +1,111 @@
+    case "SUCCESS":
+      return (
+        <div className="lab__details">
+          <div className="section">
+            <div className="section__title">{t("lab.patient")}</div>
+            <div className="section__content">
+              <div>
+                <strong>{t("lab.code")}</strong> -{" "}
+                <span>{lab?.patientCode}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.name")}</strong> - <span>{lab?.patName}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.sex")}</strong> - <span>{lab?.sex}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.age")}</strong> -{" "}
+                <span>
+                  {lab?.age} {t("lab.years")}
+                </span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.status")}</strong> -{" "}
+                <span>
+                  {lab?.inOutPatient === "O"
+                    ? t("lab.outpatient")
+                    : t("lab.inpatient")}
+                </span>
+              </div>
+            </div>
+          </div>
+          <div className="section">
+            <div className="section__title">{t("lab.exam")}</div>
+            <div className="section__content">
+              <div>
+                <strong>{t("lab.labcode")}</strong> - <span>{lab?.code}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.registrationdate")}</strong> -{" "}
+                <span>{renderDate(lab?.registrationDate ?? "")}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.examdate")}</strong> -{" "}
+                <span>{renderDate(lab?.examDate ?? "")}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.examtype")}</strong> -{" "}
+                <span>{lab?.exam?.examtype?.description}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.exam")}</strong> -{" "}
+                <span>{lab?.exam?.description}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.material")}</strong> -{" "}
+                <span>{lab?.material}</span>
+              </div>
+              |
+              <div>
+                <strong>{t("lab.result")}</strong> - <span>{lab?.result}</span>
+              </div>
+            </div>
+            {lab?.note != null && (
+              <div className="section">
+                <div className="section__title">{t("lab.note")}</div>
+                <div className="section__content_note">{lab?.note}</div>
+              </div>
+            )}
+          </div>
+        </div>
+      );
+    default:
+      return <InfoBox type="error" message={errorMessage} />;
+  }
+};
```

Comment:
![image](https://user-images.githubusercontent.com/8697550/177333367-0f8905f3-f6cd-4146-9989-1577be344db3.png)

I think that we can enhance the UI of this component with 2 inner horizontal align boxes with the Patient and the Exam information and below the note text.